docs/src/controller.md

```diff
@@ -1,123 +1,123 @@
-# Custom Pod Controller
+# Custom pod controller
 
 Kubernetes uses the
-[Controller pattern](https://kubernetes.io/docs/concepts/architecture/controller/)
+[controller pattern](https://kubernetes.io/docs/concepts/architecture/controller/)
 to align the current cluster state with the desired one.
 
 Stateful applications are usually managed with the
 [`StatefulSet`](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/)
-controller, which creates and reconciles a set of Pods built from the same
-specification, and assigns them a sticky identity.
+controller, which creates and reconciles a set of pods built from the same
+specification and assigns them a sticky identity.
 
-CloudNativePG implements its own custom controller to manage PostgreSQL
-instances, instead of relying on the `StatefulSet` controller.
+Instead of relying on the `StatefulSet` controller, CloudNativePG implements 
+its own custom controller to manage PostgreSQL instances.
 While bringing more complexity to the implementation, this design choice
 provides the operator with more flexibility on how we manage the cluster,
 while being transparent on the topology of PostgreSQL clusters.
```

Comment:
I wasn't sure who the "we" was in this case. Would this be accurate if "we" were switched to "you"? Or does EBG or someone else provide a service in managing the clusters?